### ISSUE TITLE
Kata points and solution missing even after successfully submitting the solution. Solution is present in the profile solved katas though but no points granted.

### ISSUE THREAD

**Issue by md-farhan-memon** [Codewars/codewars.com#786]
Even after kata is completed and submitted successfully, the score isn't added to my profile and neither the solution is visible on the all solutions page. It says, "There are no solutions yet for this kata."

This is my profile: https://www.codewars.com/users/md.farhanmemon
I completed this Kata: https://www.codewars.com/kata/57feb00f08d102352400026e
Solutions Page: https://www.codewars.com/kata/57feb00f08d102352400026e/solutions/ruby/all/best_practice

**Comment by tachyonlabs** [comment]
Did you try waiting a while and checking again? Sometimes your points and solution (and published katas, probably other things too) take longer than usual to show up.

**Comment by md-farhan-memon** [comment]
Yeah..it is over an hour now but still not updated....usually it shows up without delay..

**Comment by md-farhan-memon** [comment]
Heyy..a good and a bad news...
The solution is now visible but my honors didn't update..
There was a message in the stats section saying next update in 1 hour..I was hoping this will fix the issue but don't seem to be completely solved.
